Dockerfile

```diff
@@ -1,45 +1,60 @@
 FROM ruby:2.3
 
-RUN apt-get update && \
-    apt-get install --no-install-recommends -y cron && \
-    rm -rf /var/lib/apt/lists/* && \
-    apt-get clean
+RUN SUPERCRONIC_URL=https://github.com/aptible/supercronic/releases/download/v0.1.3/supercronic-linux-amd64 && \
+    SUPERCRONIC_BIN=/usr/local/bin/supercronic && \
+    SUPERCRONIC_SHA1SUM=96960ba3207756bb01e6892c978264e5362e117e && \
+    curl -fsSL -o "${SUPERCRONIC_BIN}" "${SUPERCRONIC_URL}" \
+    && echo "${SUPERCRONIC_SHA1SUM}  ${SUPERCRONIC_BIN}" | sha1sum -c - \
+    && chmod +x "${SUPERCRONIC_BIN}"
 
 ENV RAILS_ENV=production \
+    RAILS_LOG_TO_STDOUT=true \
     RAILS_SERVE_STATIC_FILES=true
 
 WORKDIR /usr/src/app
-COPY . ./
-RUN echo $SOURCE_COMMIT > REVISION
 
-RUN buildDeps='libmagic-dev' && \
+# separately from rest of the code to avoid re-running bundler when Gemfile doesn't change
+# plugins is copied completely, because COPY can't pick wildcard directories
+COPY plugins plugins/
+COPY Gemfile Gemfile.lock ./
```

Comment:
why? we create a image for RELEASE, so this does not apply here.

Reply:
When developing the `Dockerfile` I found it saved some time.